app/_how-tos/custom-deck-object-defaults.md

```diff
@@ -0,0 +1,153 @@
+---
+title: decK object defaults reference
+
+works_on:
+    - on-prem
+    - konnect
+
+tags:
```

Comment:
we decided to set the `product` as `api-ops` too for deck's how-tos, that way we could filter how-tos etc by `product` and `tool`.